opm/parser/eclipse/EclipseState/Schedule/MSW/SICD.hpp

```diff
@@ -91,33 +92,6 @@ namespace Opm {
             serializer(m_scaling_factor);
         }
 
-        template<class ICD>
-        static std::map<std::string, std::vector<std::pair<int, ICD> > >
-        fromWSEG(const DeckKeyword& wseg) {
```

Comment:
This could have been a free function in a private header to avoid the code duplication?

Reply:
ah yes, i somehow got the idea it accessed members. will improve.

Reply:
done.

Reply:
Thanks for the update - will merge when green.